README.md

```diff
@@ -159,7 +159,9 @@ const isStamp = (obj) => {
     typeof obj.compose === 'function';
 };
 
-const stamp = rtype`interface stamp ${ isStamp }`;
+const stamp = rtype(isStamp);
```

Comment:
I was just experimenting with ideas on how to set up a string representation of the type, but I think it would be better to abandon that idea for now... I think we'd need to introduce a compile step to make template strings a useful tool with this. =)